Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -3051,7 +3051,7 @@
 		93A806201E03B585008A1F26 /* JSLongRange.h in Headers */ = {isa = PBXBuildFile; fileRef = 93A8061C1E03B585008A1F26 /* JSLongRange.h */; };
 		93B0A65326CDD14100AA21E4 /* NavigatorPermissions.h in Headers */ = {isa = PBXBuildFile; fileRef = 93B0A64A26CDD13E00AA21E4 /* NavigatorPermissions.h */; };
 		93B0A65426CDD14100AA21E4 /* Permissions.h in Headers */ = {isa = PBXBuildFile; fileRef = 93B0A64B26CDD13E00AA21E4 /* Permissions.h */; };
-		93B0A65626CDD14100AA21E4 /* PermissionStatus.h in Headers */ = {isa = PBXBuildFile; fileRef = 93B0A64D26CDD13F00AA21E4 /* PermissionStatus.h */; };
+		93B0A65626CDD14100AA21E4 /* PermissionStatus.h in Headers */ = {isa = PBXBuildFile; fileRef = 93B0A64D26CDD13F00AA21E4 /* PermissionStatus.h */; settings = {ATTRIBUTES = (Private, ); }; };
```

Comment:
Does this header really need to be made private? I don't see it getting included from WebKit2.

Reply:
It's included in `WebPermissionController.cpp` since `WebPermissionController::permissionChanged` uses `WebCore::PermissionStatus::stateChanged`

Reply:
It's included in WebPermissionController.cpp since WebPermissionController::permissionChanged uses WebCore::PermissionStatus::stateChanged

Reply:
I reverted this change since `WebPermissionController.cpp` no longer includes that file after reverting some changes made to observers. Fixed!